app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflow/FavouriteWorkflowsPresenter.java

```diff
@@ -84,19 +84,18 @@ public void onNext(List<Workflow> workflowList) {
                             getMvpView().showEmptyWorkflow();
                         }
                     }
-                });
-
+                }));
     }
 
-
     public void attachSearchHandler(SearchView searchView) {
+        checkViewAttached();
         RxSearch.fromSearchView(searchView)
                 .debounce(300, TimeUnit.MILLISECONDS)
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(new Subscriber<String>() {
+                .subscribeWith(new DisposableObserver<String>() {
                     @Override
-                    public void onCompleted() {
-
+                    public void onNext(String s) {
```

Comment:
Please change `s` with specific naming convention, I am sure It is search query so It's name can be `searchQuery`